subsys/bluetooth/controller/hci/hci.c

```diff
@@ -257,6 +257,7 @@ static void host_buffer_size(struct net_buf *buf, struct net_buf **evt)
 		return;
 	}
 
+	BT_INFO("FC: host buf size: %d", acl_pkts);
```

Comment:
BT_INFO should be used very conservatively. It's basically for information that can be considered useful "almost always" regardless of what the user is testing or debugging. I don't think this qualifies. BT_INFO gets enabled whenever CONFIG_BLUETOOTH_DEBUG is enabled, whereas BT_DBG gets enabled when debug has been enabled for the specific module in question.